client_new/examples/README.md

```diff
@@ -3,13 +3,27 @@
 A few examples to explain how Stronghold works. Two types will be presented here: a simple command line interface for generating secrets, working with the store etc.
 and showcasing the p2p functionality to exchange / execute remote cryptographic procedures over the p2p interface from Stronghold.
```

Comment:
```suggestion

The following examples explain how Stronghold works. There are two sets of examples:

1) [Command-line interface examples](#command-line-interface): A simple command-line interface for generating secrets, working with the store, and other topics.

2) Examples showcasing Stronghold’s p2p functionality: Exchange and execute remote cryptographic procedures over the p2p interface from Stronghold.
```
Is the title missing for the second set? I mean the ones which are not cli. It may be a good idea to add the TOC as sublists here. Wdyt?

Reply:
@lucas-tortora yes, p2p examples coming up, as soon as the network client has been ported. The next update contains the two sections as sublists. I think that's a good idea. Also the content should reside inside the wiki as well :)